src/language/__tests__/visitor-test.ts

```diff
@@ -893,6 +894,41 @@ describe('Visitor', () => {
     ]);
   });
 
+  it('visits only the specified `Kind` in visitorKeyMap', () => {
+    const visited: Array<[string, string, ReturnType<typeof getValue>]> = [];
+
+    const visitorKeyMap: VisitorKeyMap<ASTKindToNode> = {
+      Document: ['definitions'],
+      OperationDefinition: ['name'],
+    };
+
+    const visitor: ASTVisitor = {
+      enter(node) {
+        visited.push(['enter', node.kind, getValue(node)]);
+      },
+      leave(node) {
+        visited.push(['leave', node.kind, getValue(node)]);
+      },
+    };
+
+    const exampleDocumentAST = parse(/* GraphQL */ `
```

Comment:
We don't use `/* GraphQL */` tag in our source code.
All tests in this file name their variable simply `ast`.
Always try to match the style of another test.